broker_test.go

```diff
@@ -362,3 +362,24 @@ func TestSuccessThreshold(t *testing.T) {
 		t.Fatalf("expected successThreshold error")
 	}
 }
+
+func TestSuccessThresholdSinks(t *testing.T) {
+	b := NewBroker()
+
+	err := b.SetSuccessThresholdSinks("t", 2)
+	if err != nil {
+		t.Fatal(err)
+	}
+	g, ok := b.graphs["t"]
+	if !ok {
+		t.Fatalf("expected graph for eventType")
+	}
+	if g.successThresholdSinks != 2 {
+		t.Fatalf("expected successThresholdSinks %d, not %d", g.successThresholdSinks, 2)
```

Comment:
```suggestion
		t.Fatalf("expected successThresholdSinks %d, not %d", 2, g.successThresholdSinks)
```

Reply:
This is my bad as I just copy pasta the test above, I think maybe the order of the args is the wrong way around? 

If so, the L.357 also needs to be adjusted:

`t.Fatalf("expected successThreshold %d, not %d", 2, g.successThreshold)`

Reply:
updated both tests.